src/main/java/org/zalando/fahrschein/ClientHttpRequestFactoryHelper.java

```diff
@@ -0,0 +1,34 @@
+package org.zalando.fahrschein;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.client.ClientHttpRequestFactory;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+
+public class ClientHttpRequestFactoryHelper {
```

Comment:
> I don't get that class and wrapping. :worried: Can you tell me, what's the purpose?

The idea was to automatically wrap a request factory so it does authorization, problem handling and work around an issue with apache http client. For problems and authorization there is a better idea in issue #15 (using a builder for the nakadi client), the issue with apache http client can be worked around outside the library as described in #17.